### PR TITLE
Update Twitter icon, strings and update icons

### LINKED ISSUE #324: Update Twitter icons and translations
With new name of Twitter, we need to update translations and icons used in AboutFragment.

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
- Fix #324
- Renamed some icons
- Updated and optimized icon path svg
- Some icons are now rounder
- Deleted unused icons

| Before | After |
|---|---|
| |  |

### CODE REVIEW

File: app/src/main/res/xml/about_preference.xml [hunk 1]
```diff
@@ -33,11 +33,11 @@
     <Preference
         android:icon="@drawable/ic_mastodon"
         android:key="mastodon"
-        android:summary="@string/twitter_desc"
+        android:summary="@string/X_desc"
         android:title="@string/mastodon" />
     <Preference
-        android:icon="@drawable/ic_twitter"
+        android:icon="@drawable/ic_x"
         android:key="twitter"
```
Comment (codeurimpulsif) [inline app/src/main/res/xml/about_preference.xml:40]: Don't you want to rename the `android:key` too?

**Review by codeurimpulsif — CHANGES_REQUESTED** [review]

**Comment by codeurimpulsif** [comment]
Also, do you think it's a good idea to change this now? Everyone seem to still refer to this social network as "Twitter"

**Comment by Jean-BaptisteC** [comment]
Next version is not planned for now (September or October, maybe).
I think a most of users know new name of Twitter, if you want, i keep new icon with older name ?

**Comment by codeurimpulsif** [comment]
I don't know, maybe we can speak about this at the next meeting?

**Comment by Jean-BaptisteC** [comment]
I have found a better way, I suggest for Twitter string `X (Twitter)`

**Review by xakan — APPROVED** [review]

**Review by codeurimpulsif — APPROVED** [review]

**Pull request merged**